src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDriverAdapter.java

```diff
@@ -26,8 +24,9 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
+		edu.kis.powp.jobs2d.features.DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
Direct coupling with DrawerFeature class is unnecessary. This is not a good way to get rid of the import, because it does not remove the dependency.